### DISCUSSION TITLE
Drum samples and YAML files if you want them

### DISCUSSION THREAD

**Discussion by briwil** [FundamentalFrequency/.github#51] [Show and tell]
I just uploaded 72 drum kits, drum samples mostly from Logic and my MC-707, along with all the YAML files in their respective folders, so they should be good to just drop in your drum_kits folder. (Big thank you to @caschy for his great python script that significantly sped up the kit making process).
If anyone wants them-

Edit: July 18,2022- new link after fixing issues listed below-
https://www.dropbox.com/s/32358jgi2qyq1q0/drum_kits_220718.zip?dl=0

**Comment by kuffcakes** [comment]
These worked on my LMN-3! I couldn't get the drum samples I loaded to play but yours work fine. Thanks for uploading these.

**Reply by briwil** [reply]
Oh nice! I haven't had a chance to check them all- I had one that had a YAML that was missing a number and it kept causing the LMN-3 to crash each time I would just scroll over the kit name, and the kits don't seem to be arranged in any particular order, so it took a while to figure out which file was the bad one... Anyway I'm hoping they are all correct.

**Comment by raychmond** [comment]
@briwil Thanks a lot for sharing these. I checked many of your drum kits. Most of them work perfectly. 
I almost finish building my LMN-3 and try to find some drum kits and samples. You just saved my life. Really appreciate it!

**Reply by briwil** [reply]
Great- if you see ones that are messed up let me know so I can update please.

**Comment by raychmond** [comment]
@briwil 
I checked all your drum kits. 73 kits showed up in the menu. Only 72 folders are found in the archive. I found the following errors.

Thank you!

*Updated July 18, 2022 Thanks @k-kalle for finding this error.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/paull/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/paull/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Name | Comment
-- | --
MC707_90s_Kit_1 | It contains another   folder:East_Bay_Kit
Linn_Drums | Windows Media Player   can't even play .wav files
MC707_TR-626_3 | Can't find folder   with this name. It may be in MC707_Studio_Kit_1 folder*
MC707_New_Wave_Kit_3 | YAML shows .WAV, the   file name ext. is aif
East_Bay_Kit | Can't find folder   with this name. It may be in MC707_90s_Kit_1 folder
Speak_Easy_Kit | 08,09,10 don't have   sound. Rest are fine.
MC707_Room_Kit_1 | 17 doesn't have   sound. Windows Media Player can't play the track. Rest are fine.



</body>

</html>

**Reply by briwil** [reply]
Wow, a lot of mistakes actually. Thank you for finding them!

**Reply by raychmond** [reply]
Actually not too many. You've done a great work. A few tracks are not working in a few kits.
Do you happen to have other samples rather than the drum kit? Thank you!

**Reply by briwil** [reply]
I do have quite a bit of samples, I've been sampling instruments recently for my Synthstrom Deluge so my library has been growing, though I have yet to add any of them to them LMN-3. @stonepreston am I correct to think the LMN-3 doesn't do multisampling, just one file loaded at a time? If so, should the file always be C note?
Once I get my info straight I can start putting together some samples to share.

**Reply by raychmond** [reply]
Thank you @briwil @stonepreston 
I hope I could contribute more to the community. I'm neither a developer nor a musician. But at least I can do some testing or provide some information about the parts.

**Reply by briwil** [reply]
Haha, you're neither a developer nor a musician?... what are you doing here then??

Either way, glad you are here to help!

**Reply by raychmond** [reply]
I highly respect the open source developers like you guys. With my limited knowledge in this field (I'm an actuary), I'm trying to contribute while learning the knowledge of engineering, software development and music.

**Reply by k-kalle** [reply]
Thanks a lot for the package! Noticed one more error: MC707_Studio_Kit_1 folder has subfolder MC707_TR-626_3

**Reply by raychmond** [reply]
Thanks @k-kalle. I have added this in the above chart, so @briwil or other people can notice and fix it.

**Reply by briwil** [reply]
Thanks, just updated link.

**Reply by hankydotorg** [reply]
Hi @raychmond ! Do you happen to have these samples? I think a lot of us are hoping to get access again :)

**Comment by PupB** [comment]
@briwil Thanks for sharing these drum samples a while back! I was really looking forward to checking it out, but it seems like the Dropbox link is no longer working. Do you happen to have another copy of it that you could share? I'd really appreciate it.

**Reply by smczech** [reply]
@briwil I would also love a copy of these.

**Reply by briwil** [reply]
Sorry I don't think I have them anymore.

**Reply by smczech** [reply]
Darn! Does anyone on here have them?

**Reply by hankydotorg** [reply]
Yes please - am also looking for this. <3

**Comment by raychmond** [comment]
I might still have them. Can you please remind me where these files are saved on raspberry pi. I can't fine them on my computer, but the LMN-3 is still alive.
    On Monday, July 8, 2024 at 01:42:23 p.m. EDT, hankydotorg ***@***.***> wrote:  
 
 


Hi @raychmond ! Do you happen to have these samples? I think a lot of us are hoping to get access again :)

**Reply by PupB** [reply]
~/.config/LMN-3/drum_kits is where they should be saved.

**Comment by raychmond** [comment]
Thanks! I found them. I will need to copy them back to my computer and share with you.
I’ll do this tomorrow.

Sent from Yahoo Mail for iPhone


On Monday, July 8, 2024, 8:47 PM, PupB ***@***.***> wrote:




~/.config/LMN-3/drum_kits is where they should be saved.

**Comment by raychmond** [comment]
drum_kits

Please let me know if the link doesn't work.
Thanks,    On Monday, July 8, 2024 at 11:29:41 p.m. EDT, Paul Luo ***@***.***> wrote:  
 
 Thanks! I found them. I will need to copy them back to my computer and share with you.
I’ll do this tomorrow.

Sent from Yahoo Mail for iPhone


On Monday, July 8, 2024, 8:47 PM, PupB ***@***.***> wrote:




~/.config/LMN-3/drum_kits is where they should be saved.

**Reply by hankydotorg** [reply]
Yay!  The link doesn't work for me... Please do share again! I'm so excited!

**Comment by smczech** [comment]
I’m excited too!Sent from my  iPhone 15 Pro Max (256GB)On Jul 9, 2024, at 9:34 AM, hankydotorg ***@***.***> wrote:﻿
Yay!  The link doesn't work for me... Please do share again! I'm so excited!

**Comment by raychmond** [comment]
drum_kits

Please try a couple of more times. I tried the link on my other computer. It didn't work. But after a few more clicks, it worked.
Thanks,
    On Tuesday, July 9, 2024 at 10:44:16 a.m. EDT, smczech ***@***.***> wrote:  
 
 

I’m excited too!Sent from my  iPhone 15 Pro Max (256GB)On Jul 9, 2024, at 9:34 AM, hankydotorg ***@***.***> wrote:﻿
Yay! The link doesn't work for me... Please do share again! I'm so excited!

**Reply by hankydotorg** [reply]
Hi @raychmond ! Are you able to paste the hyperlink directly?
Somehow, when the post gets published as a comment, it just says "drum_kits" but without a link. 
I'm hoping that a direct hyperlink may help. 
Alternatively, are we able to find you on the Discord server?

Thank you for your public service!

**Reply by raychmond** [reply]
https://1drv.ms/f/s!Ak3OXcjNGLBXicov2xFdzFTFDQswDw?e=4VwFiY

**Reply by raychmond** [reply]
Hopefully this works. 
Sorry, I wasn't logged in github to post. I was using the emails to reply. That may be the issue that you didn't see the link correctly.

**Reply by hankydotorg** [reply]
This is amazing! Thank you, busy downloading now! I wonder if we could find a way to have some communal sample-sharing. I've definitely scrounged together a couple of samples I would love to share. Thank you!

**Reply by raychmond** [reply]
Great idea! It may encourage me to pick up my device and have some fun.

**Reply by hankydotorg** [reply]
Amazing! I'm doing a little research on what may be the best platform. Google Drive? OneDrive? Or here on Github? I'm actually not very familiar with Github as I'm not a programmer, but can learn pretty fast.  Would you be interested in co-thinking a way to invite people to share samples? Let me know! (No pressure)

**Reply by raychmond** [reply]
Awesome, I'm not a programmer either. Maybe we can start from here by posting the shared links in this post, or open a new post. The drum sample set is my only collection so far. But I can do some searches to find more and share here.

**Reply by hankydotorg** [reply]
Great, thanks @raychmond - I've just started a new discussion post here: https://github.com/orgs/FundamentalFrequency/discussions/119

I'm busy uploading all of my samples currently and will post there asap! please do post your onedrive link there as well with @briwil 's drum samples :D

**Reply by PupB** [reply]
Thank you for sharing @raychmond !!!
